PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -391,6 +402,49 @@ fi
 
 # log_Msg "POSIXLY_CORRECT="${POSIXLY_CORRECT} #NOT DEFINED ANYWHERE ELSE DO WE NEED THIS? 
 
+# default parameters
+SPECIES=`opts_DefaultOpt $SPECIES Human`
+RunMode=`opts_DefaultOpt $RunMode 1`
+TruePatientPosition=`opts_DefaultOpt $TruePatientPosition HFS`  
+ScannerPatientPosition=`opts_DefaultOpt $ScannerPatientPosition HFS` 
+betcenter=`opts_DefaultOpt $betcenter 45,55,39`        
+betradius=`opts_DefaultOpt $betradius 75` 
+betfraction=`opts_DefaultOpt $betfraction 0.3` 
+bettop2center=`opts_DefaultOpt $bettop2center 86` 
+BrainExtract=`opts_DefaultOpt $BrainExtract INVIVO` 
```

Comment:
`opts_DefaultOpt` is from the old options parsing, argument defaults should now be passed to `opts_AddOptional`.  However, none of these appear to be arguments, and many appear to be specific to your data (suggesting that the pipeline shouldn't have default values for `betcenter` or `betradius`.

Additionally, under `set -eu`, the fact that none of these variables were defined before these lines implies that unless they were exported before running this script, it will abort at these lines.  We don't use exporting for anything other than tool install locations, so I think the ones that the user needs to change for processing should be turned into `opts_AddMandatory` and `opts_AddOptional`, and all these lines removed.

In particular, `RunMode` is hardcoded under various `CustomBrain` settings, so it probably doesn't need the ability to be set to something else without editing the pipeline script.